recipes/audioreach-pipewire-plugin/audioreach-pipewire-plugin_git.bb

```diff
@@ -0,0 +1,26 @@
+inherit autotools pkgconfig
+SUMMARY = "Pipewire pal plugins"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=91f36d19ef812a054b22d918288de2b5"
+
+SRCREV = "${AUTOREV}"
```

Comment:
Please stop using AUTOREV, it's a pain to support. Use fixed 'known good' revisions and optionally provide an include snippet for developers to bump revisions to `${AUTOREV}`. Generally, your layer should build successfully with `BB_NO_NETWORK=1`.